model/src/main/java/com/ge/predix/acs/rest/AttributeAdapterConnection.java

```diff
@@ -56,6 +65,7 @@ public void setUaaClientId(final String uaaClientId) {
         this.uaaClientId = uaaClientId;
     }
 
+    @ApiModelProperty(value = "A secret for an OAuth client for this adapter", required = true)
```

Comment:
Might be more clearly worded as "OAuth client secret used to obtain an access token for this adapter"?